conf/log4j.properties

```diff
@@ -34,6 +37,14 @@ log4j.appender.MASTER_LOGGER.MaxBackupIndex=100
 log4j.appender.MASTER_LOGGER.layout=org.apache.log4j.PatternLayout
 log4j.appender.MASTER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%n
 
+# Remote appender for Master
+log4j.appender.REMOTE_MASTER_LOGGER=org.apache.log4j.net.SocketAppender
+log4j.appender.REMOTE_MASTER_LOGGER.Port=${alluxio.logserver.port}
+log4j.appender.REMOTE_MASTER_LOGGER.RemoteHost=${alluxio.master.hostname}
```

Comment:
this doesn't work in fault-tolerant mode since there is no single master hostname. I think it's better to let the user configure this file if they want to set up a remote appender.

Reply:
Done.